src/Microsoft.OpenApi.OData.Reader/Vocabulary/Core/CoreConstants.cs

```diff
@@ -0,0 +1,66 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -----------------------------------------------------------
+
+namespace Microsoft.OpenApi.OData.Vocabulary.Core
+{
+    /// <summary>
+    /// Constants for the Core vocabulary
+    /// </summary>
+    internal class CoreConstants
+    {
+        /// <summary>
+        /// Org.OData.Core.V1.Links
+        /// </summary>
+        public const string Links = "Org.OData.Core.V1.Links";
+
+        /// <summary>
+        /// Org.OData.Core.V1.Revisions
+        /// </summary>
+        public const string Revisions = "Org.OData.Core.V1.Revisions";
+
+        /// <summary>
+        /// Link relation types.
+        /// </summary>
+        public static class LinkRel
```

Comment:
instead of having hardcoded constants to Microsoft Graph, I'd have a configuration object on the settings, which default to those values. This way people using it for other APIs can always change the links.

Reply:
correct